linked_list/LinkedList.cpp

```diff
@@ -0,0 +1,423 @@
+/*
+ * C++ Program to Implement Singly Linked List
+ */
+#include<iostream>
+#include<cstdio>
+#include<cstdlib>
+using namespace std;
+/*
+ * Node Declaration
+ */
+struct node
+{
+    int info;
+    struct node *next;
+}*start;
+
+/*
+ * Class Declaration
+ */
+class single_llist
+{
```

Comment:
Please use standard cpp indentations throughout the code, and be consistent with the spacings.